python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -201,6 +201,9 @@ def merge_resources(self, resources_dict: Mapping[str, Any]) -> "SensorEvaluatio
         check.invariant(
             self._resources is None, "Cannot merge resources in context that has been initialized."
         )
+        from dagster._core.execution.build_resources import wrap_resources_for_execution
+
+        wrapped_resources_defs = wrap_resources_for_execution(resources_dict)
```

Comment:
same comment here that having the variable `wrapped_resources_defs` actually obscures things